### DISCUSSION TITLE
Customize Border Width and Color

### DISCUSSION THREAD

**Discussion by roopsam** [damongolding/immich-kiosk#181] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to be able to customize the border width for screens where you don’t want to waste that space, it would also be nice to set the border color to black or white in the config depending on the screen bezel.

**Comment by damongolding** [comment]
I have added a `frameless` option in 0.13.0 that will remove the border and rounded corners to maximise the image to screen ratio

**Comment by roopsam** [comment]
That's awesome! Thanks for all the work you have put in, I really love using Immich Kiosk, and it was fairly easy to set up.
